src/oscar/templates/oscar/dashboard/offers/offer_list.html

```diff
@@ -95,12 +103,16 @@ <h1>{% trans "Offers" %}</h1>
                                         <a class="dropdown-item" href="{% url 'dashboard:offer-detail' pk=offer.pk %}">
                                             {% trans "Stats" %}
                                         </a>
+                                        {% if not offer.is_voucher_offer_type %}
                                         <a class="dropdown-item" href="{% url 'dashboard:offer-metadata' pk=offer.pk %}">
                                             {% trans "Edit" %}
                                         </a>
+                                        {% endif %}
+                                        {% if not offer.vouchers.exists %}
```

Comment:
Note that while this option is hidden here, it is still accessible from the offer detail view (delete button at the top right), and if you use that you can still delete the offer. I think the delete view needs validation to prevent this happening.